sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2956,6 +2956,23 @@ object GpuOverrides extends Logging {
               target: Option[Expression] = None): GpuExpression =
             GpuStringTrimRight(column, target)
         }),
+    expr[StringTranslate](
+      "StringTranslate operator",
+      ExprChecks.projectOnly(TypeSig.STRING, TypeSig.STRING,
+        Seq(ParamCheck("input", TypeSig.STRING, TypeSig.STRING),
+          ParamCheck("from", TypeSig.lit(TypeEnum.STRING), TypeSig.STRING),
+          ParamCheck("to", TypeSig.lit(TypeEnum.STRING), TypeSig.STRING))),
+      (in, conf, p, r) => new TernaryExprMeta[StringTranslate](in, conf, p, r) {
+        override def convertToGpu(
+            input: Expression,
+            from: Expression,
+            to: Expression): GpuExpression =
+          // Compatible with Spark 3.2+. Only in Spark 3.2+ does translate() support
+          // unicode characters with code point >= U+10000. Since it was a bug fix,
+          // we have kept it in all versions.
+          // See https://issues.apache.org/jira/browse/SPARK-34094.
+          GpuStringTranslate(input, from, to)
+      }),
```

Comment:
You should switch this to using `incompat` to note the compatibility issue on the generated online documentation. Then regenerate the documentation. Something like the following

```suggestion

          GpuStringTranslate(input, from, to)
      }).incompat("The GPU supports all unicode code points. In Spark versions < 3.2.0 translate() does not " +  "support unicode characters with code point < U+10000 (See SPARK-34094)"),
```

Reply:
Thank you! Changes updated.